setup.py

```diff
@@ -27,6 +28,12 @@
     'pytest',
 ]
 
+# WARNING: This imposes limitations on test/requirements.txt such that the
+# full Pip syntax is not supported. See also
+# <http://stackoverflow.com/questions/14399534/>.
+with open(os.path.join('test', 'requirements.txt')) as f:
```

Comment:
Okay, just for the sake of my own edification, _where_ (tf) is that documented? 😳